core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/executionrequest/ExecutionRequestControllerTest.java

```diff
@@ -102,58 +104,63 @@ public void testPagination() throws Exception {
 
     @Test
     public void getAllNoResultTest() throws Exception {
-        List<ExecutionRequest> executionRequests = new ArrayList<>();
-        given(executionRequestService.getAll(10,0,null,null,null,null,null,null)).willReturn(executionRequests);
+        List<ExecutionRequestDto> executionRequests = new ArrayList<>();
+        given(executionRequestService.getAll()).willReturn(executionRequests);
         mvc.perform(get("/execution-requests?limit=1&pageNumber=1")
                 .contentType(MediaType.APPLICATION_JSON)
         ).andExpect(status().isOk());
     }
 
```

Comment:
At this moment, this test file (not just this test) doesn't test anything.

In my opinion, controller tests are used to
- check if it calls the service the way we expect it to. (For example: is a Pageable object rightly created and passed)
- check if it makes the data transformation it should.
- check if the return Dto are consistent regarding expectations.

Right now, all tests are just checking if a 200 is received, which should always be true as we just call the mocked service.